src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -228,30 +233,29 @@ const $Row = styled.span`
   --item-height: 1.25rem;
 `;
 
-const $Label = styled.div<{ warning?: boolean; hasOrders: boolean }>`
-  align-items: center;
-  border: solid var(--border-width) var(--color-border);
-  border-radius: 0.5em;
-  display: flex;
-  font: var(--font-tiny-book);
-  height: var(--item-height);
-  padding: 0 0.25rem;
+const $TriggerButton = styled(Button)<{ warning?: boolean; hasOrders: boolean }>`
+  --button-backgroundColor: transparent;
+  --button-border: solid var(--border-width) var(--color-border);
+  --button-font: var(--font-tiny-book);
+  --button-height: var(--item-height);
+  --button-padding: 0 0.25rem;
+  --button-textColor: var(--color-text-0);
 
   ${({ warning }) =>
     warning &&
     css`
-      background-color: var(--color-warning);
-      color: var(--color-black);
+      --button-textColor: var(--color-black);
+      --button-backgroundColor: var(--color-warning);
     `}
 
   ${({ hasOrders }) =>
     hasOrders
       ? css`
-          color: var(--color-text-1);
-          background-color: var(--color-layer-4);
+          --button-textColor: var(--color-text-1);
+          --button-backgroundColor: var(--color-layer-4);
         `
       : css`
-          color: var(--color-text-0);
+          --button-hover-textColor: var(--color-text-1);
```

Comment:
did this because otherwise the hover is very faint (brightness filter doesn't do that much) and not very discoverable.